### ISSUE TITLE
Docs feedback: /docs/sources/clients/promtail/stages/regex.md

### ISSUE THREAD

**Issue by danfoxley** [grafana/loki#9788]
With pattern parser in Loki 2.3 and this YouTube video highlighting it over regex, shouldn’t it get a mention on this page?

**Comment by JStickler** [comment]
@danfoxley Did you intend to link to a YouTube video?

**Comment by GrafanaWriter** [comment]
@JStickler  - should this be moved to Blocked?

**Comment by danfoxley** [comment]
Yes, duh forgot the link!  Your employee is showing Pattern Parser and using it instead of regex, or is at least highlighting the use of Pattern Parser over the bother of regex in Promtail.  I'm suggesting it be highlighted as an option in the document.

https://www.youtube.com/watch?v=zIdEVNA6YTI

**Comment by JStickler** [comment]
Promtail is end of life (EOL) as of  March 2, 2026.  No future support or updates will be provided. All future feature development will occur in [Grafana Alloy](https://github.com/grafana/alloy).
